client/app/utils/jsonSchema/userOperator.ts

```diff
@@ -0,0 +1,379 @@
+import { RJSFSchema } from "@rjsf/utils";
+
+export const userOperatorSchema: RJSFSchema = {
+  type: "object",
+  required: [
+    "first_name",
+    "last_name",
+    "position_title",
+    "street_address",
+    "municipality",
+    "postal_code",
+    "email",
+    "phone_number",
+    "province",
+    "role",
+    // "file", temporary handling of many-to-many fields, will be addressed in #138
+    "legal_name",
+    "business_structure",
+    "physical_street_address",
+    "physical_municipality",
+    "physical_province",
+    "physical_postal_code",
+    "mailing_street_address",
+    "mailing_municipality",
+    "mailing_province",
+    "mailing_postal_code",
+    "operator_has_parent_company",
+  ],
+  properties: {
+    step_1: {
+      //Not an actual field in the db - this is just to make the form look like the wireframes
+      type: "object",
+      title: "Step 1. User Information",
+      readOnly: true,
+    },
+    first_name: { type: "string", title: "First Name" },
+    last_name: { type: "string", title: "Last Name" },
+    position_title: { type: "string", title: "Position Title" },
+    street_address: { type: "string", title: "Mailing Address" },
+    municipality: { type: "string", title: "Municipality" },
+    postal_code: { type: "string", title: "Postal Code" },
+    email: { type: "string", title: "Email Address" },
+    phone_number: { type: "string", title: "Phone Number" },
+    province: { type: "string", title: "Province" },
```

Comment:
There's an enum dropdown for allowed provinces you can grab from the operations schema

Reply:
Thanks for this 🙏